src/apps/moola/debtTokens.ts

```diff
@@ -0,0 +1,76 @@
+import { Address } from "viem"
+
+export interface DebtTokenDefinition {
+    baseTokenAddress: Address
+    debtTokenAddress: Address
+    imageUrl: string
+    title: string
+    description: string
+  }
+  
+export const MOOLA_DEBT_TOKENS: DebtTokenDefinition[] = [
+    {
+      baseTokenAddress: '0xd8763cba276a3738e6de85b4b3bf5fded6d6ca73',
+      debtTokenAddress: '0xfb6c830c13d8322b31b282ef1fe85cbb669d9ae8',
+      imageUrl:
+        'https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/mcEUR.png',
```

Comment:
I couldn't find a better image for these tokens, I hope this is ok for this PR.
But I think we should use different images for these, maybe adding some red border (to indicate a debt) to those images? Should I ask Reed to help us? Or maybe someone knows something that we can already use here.

Reply:
For now you can use the Moola image for all of them. Like we do for Ubeswap. This may evolve in the future but is what Reed wanted.